### DISCUSSION TITLE
Is there a guide on how to use this and deploy themes?

### DISCUSSION THREAD

**Discussion by Simon-ProjectPartners** [WebDevStudios/wd_s#772] [Q&A]
I'd like to transition from traditional _s, but the structure of this is a bit different. I'm a bit lost as it's all a bit overwhelming coming from vanilla underscores. Is there some more documentation apart from this github, or any guides you'd recommend?

Do I edit the scss and js in the `src` folder? Can I disable prettier

For example, I managed to get this up and running and made some test changes to some JS files. Prettier keeps screaming errors at me, even though the modifications were quite simple and follow a modern syntax.

How do I actually deploy the theme with production ready assets?

Sorry if this doesn't make much sense, just trying to get used to this :)

**Comment by shikkaba** [comment]
There are also files that are unclear what they are. Like what are all the scaffolding files? What do they do? They look like individual theme pieces, but then they are surrounded by sections.

**Comment by coreymcollins** [comment]
@Simon-ProjectPartners I don't think anyone ever saw your original message, sorry about that! The theme has changed quite a bit from October of 2021, but the basics still persist – the `/src/` folders contain your Sass, JS, and font files (if you're using any) which you will add to and edit as needed. We use [WordPress Coding Standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/), so Prettier will now follow those standards unless you have settings in your editor to override them.

When you run `npm run build`, all of your compiles JS and CSS will be built in the `/build` directory. These are the scripts that are enqueued on the frontend, so you need to deploy the `/build` directory with the rest of your theme files (excluding things like `/node-modules/` and other files/directories in `.gitignore`).

@shikkaba – we include a Scaffolding page templates which displays all of the base theme markup like headings, paragraph text, forms, image alignments, etc. These individual templates are in the `/template-parts/scaffolding/` directory and called by the `template-scaffolding.php` template file in the theme. You don't need to use the scaffolding template or scaffolding files at all; these are just there to give a one-page view of the global styles of the theme.

**Reply by shikkaba** [reply]
Thank you @coreymcollins

**Comment by shikkaba** [comment]
@coreymcollins So, after you've completed your work, and you've run build, are you supposed to manually grab all the php files and then use the build folder? The css and js has a bunch of webpack related stuff in it, and I really don't understand how this works, or if it'll be okay after. Could you explain the process of what you use when you finish compiling?

This is honestly my first time using Webpack and I've figured out quite a bit, but I'm terrified I'm messing up.
